local/launch_infrastructure.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# Copyright (c) 2024 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer ISST)
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+    BACKENDIMAGENAME="puris-backend"
+    BACKENDIMAGETAG="dev"
+
+    if docker images | grep -q "${BACKENDIMAGENAME}\s*${BACKENDIMAGETAG}"; then
```

Comment:
Done, though the highest available versioned number on docker hub currently is 2.0.2. Will use that for now. I'd rather not use the "latest" - tag, since that might lead to unexpected things, which we don't want to handle in a training session.

Reply:
You're right here. Let's stick to 2.0.2